packages/location-management/src/components/LocationUnitGroupView/index.tsx

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import React, { ChangeEvent, useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Row, Col, Menu, Dropdown, Button, Divider, Input } from 'antd';
+import { SettingOutlined, PlusOutlined } from '@ant-design/icons';
+import LocationUnitGroupDetail, { LocationUnitGroupDetailProps } from '../LocationUnitGroupDetail';
+import { SearchOutlined } from '@ant-design/icons';
+import { useDispatch, useSelector } from 'react-redux';
+import { OpenSRPService } from '@opensrp/server-service';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import reducer, {
+  fetchLocationUnitGroups,
+  getLocationUnitGroupsArray,
+  LocationUnitGroup,
+  reducerName,
+} from '../../ducks/location-unit-groups';
+import { getAccessToken } from '@onaio/session-reducer';
+import { LOCATION_UNIT_GROUP_ALL, URL_LOCATION_UNIT_GROUP_ADD } from '../../constants';
+import { API_BASE_URL } from '../../configs/env';
```

Comment:
Let's get the base URL from props

Reply:
@haseemVD have created a separate issue to deal with it .. it will be solved separately

Reply:
Created separate [issue](https://github.com/OpenSRP/web/issues/256) for this